### PR TITLE
Adding few more Gin APIs

### PR DESCRIPTION
**Pull request by dbhatnagar-px**
This PR adds few more GIn APIs:
1. Fetches and Parses Pxctl status output
2. Validates if Px is installed or not
3. Fetches Px Version

### CODE REVIEW

**Review by lsrinivas-pure — APPROVED** [review]
LGTM

File: apiServer/pxone/utils/torpedoutils.go [hunk 1]
```diff
@@ -177,3 +178,110 @@ func ScheduleAppsAndValidate(c *gin.Context) {
 		})
 	}
 }
+
+// GetPxVersion This function returns the current Px Version in the Target Cluster
+func GetPxVersion(c *gin.Context) {
+	if !checkTorpedoInit(c) {
```
Comment (mkoppal-px) [inline apiServer/pxone/utils/torpedoutils.go:184]: What if there is an error in the function `checkTorpedoInit`?
Also how does gin handle when there is nothing returned? As in, here we are not returning either `StatusOK` or `StatusInternalServerError` ?

Comment (dbhatnagar-px) [reply apiServer/pxone/utils/torpedoutils.go:184]: So, checkTorpedoInit() is InitInstance(). I was also having similar doubt but nowhere in our other test code we are checking for any errors in Initinstance(). That's the reason this method is also similar. 

Regarding not returning anything here, you are right. This seems to be a miss from my end. I should be adding a StatusInternalServerError to gin Context and then doing a null return. I will do this - thanks for catching this.

Comment (dbhatnagar-px) [reply apiServer/pxone/utils/torpedoutils.go:202]: Done. Thanks for catching this.

**Review by mkoppal-px — APPROVED** [review]
LGTM

**Pull request merged**